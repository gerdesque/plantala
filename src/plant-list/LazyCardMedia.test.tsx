import React from 'react';
import {createShallow } from '@material-ui/core/test-utils';
import { ShallowWrapper } from 'enzyme';
import LazyCardMedia from './LazyCardMedia';
import { CardMedia } from '@material-ui/core';


describe('LazyCardMedia', () => {

  let shallow: any;
  let wrapper: ShallowWrapper;
  const props = { className: 'classes.plantMedia', image: 'a.png', title: 'Kardamom' }

  beforeEach(() => {
    shallow = createShallow();
    wrapper = shallow(<LazyCardMedia {...props} />)
  })

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());

  it('should render no <CardMedia /> component', () => {
    expect(wrapper.find(CardMedia)).toHaveLength(0);
  });

  it('should render no <div /> component', () => {
    expect(wrapper.find('div')).toHaveLength(1);
  });
});
