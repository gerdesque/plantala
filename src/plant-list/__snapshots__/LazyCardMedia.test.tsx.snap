// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyCardMedia should render correctly 1`] = `
<div
  className="classes.plantMedia"
/>
`;
